pkg/cli/admin/release/extract.go

```diff
@@ -377,11 +404,10 @@ func (o *ExtractOptions) Run(ctx context.Context) error {
 					return fmt.Errorf("unrecognized platform for CredentialsRequests: %q", *inclusionConfig.Platform)
 				}
 			}
-			include = newIncluder(inclusionConfig)
 		}
 
-		tarEntryCallbacks = append(tarEntryCallbacks, func(hdr *tar.Header, _ extract.LayerInfo, r io.Reader) (bool, error) {
-			if hdr.Name == "image-references" && !o.CredentialsRequests {
+		manifestsCallbacks = append(manifestsCallbacks, func(name string, ms []manifest.Manifest, r io.Reader, enabled []configv1.ClusterVersionCapability) (cont bool, err error) {
```

Comment:
It would be helpful for me if each append to `manifestCallback` had a brief comment that said what the callback is doing

Reply:
`tarEntryCallbacks` is removed in https://github.com/openshift/oc/pull/2048/files